lib/packwerk/parsers/parser_interface.rb

```diff
@@ -0,0 +1,20 @@
+# typed: strict
+# frozen_string_literal: true
+
+module Packwerk
+  module Parsers
+    module ParserInterface
+      extend T::Helpers
+      extend T::Sig
+
+      # So Sorbet knows `nil?` exists
+      include Kernel
```

Comment:
We should not be including Kernel. The right way to fix this is with `requires_ancestor`